README.md

```diff
@@ -1 +1,69 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/praizerema)
+- Twitter: [@twitterhandle](https://twitter.com/ayodelepraisego)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/praizerema)
```

Comment:
- [ ] Instead of this code you should add your own information like the next sample (add you information) to make it more professional 😉 : 
```
## Author

👤 **Dante Alonso**

- GitHub: [@DanteAlonsoHT](URL-github-profile)
- Twitter: [@dante_dante1](URL-twiiter-profile)
- LinkedIn: [Dante Alonso](URL-linkedin-profile)
```